.gitignore

```diff
@@ -1,52 +1,39 @@
 # dependencies
-node_modules
+node_modules/
 
 # coverage
 coverage
 .nyc_output
 
+.pytest_cache/
+
 # production
-build
+build/
+dist/
+*.egg-info
+
+# Environments
+venv/
```

Comment:
I wouldn't put this in the .gitignore since it's very user-specific.  I would instead put this in your private .gitignore.

Reply:
Haha! This is actually the whole reason for me improving the `.gitignore` file. Specifically, because @colinmegill didn't like that setting up the cellxgene venv from the quick start in the readme screwed up his git status. I'm happy to delete the venv/ though.

Reply:
Would recommend setting up `.git/info/exclude` (yeah, the UX is awful).